src/main.py

```
@@ -193,39 +198,64 @@
+
+        # Ensure that not more than 10 media files are processed
+        if len(media_path) > 10:
+            debug("Too many media files to process, enabling throttle. Amount: %s", len(media_path))
+            THROTTLE = True
+
+        for pathobj in media_path:
+
+            # Create a lists of video and picture paths
+            if pathobj.endswith(".mp4"):
+                # do not process if video is too long
+                if is_video_duration_over_limits(video):
```

Ruff (0.8.2)

Undefined name `video`

(F821)